mmv1/compiler.rb

```diff
@@ -142,32 +149,49 @@
 # rubocop:disable Metrics/BlockLength
 all_product_files.each do |product_name|
   product_override_path = ''
-  product_override_path = File.join(override_dir, product_name, 'api.yaml') if override_dir
-  product_yaml_path = File.join(product_name, 'api.yaml')
+  product_override_path = File.join(override_dir, product_name, 'product.yaml') if override_dir
```

Comment:
Have you tested an override path? I don't _think_ there's anything that'd need to change, but if you haven't, we'll want to ensure that works before rolling forward with products w/ EAP features.

Reply:
good point. I wasn't sure it was still used anywhere so I haven't tested. I'll give that a go

Reply:
confirmed it works if the override directory is also updated at the same time with the new format. I'll make the appropriate change in EAP